src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    get add() {
+      return (a, b) => a + b;
+    },
```

Comment:
```suggestion
    add(a, b) {
      return a + b;
    },
```
you can use functions instead of getters

Reply:
i already understand it looking at ur solution, but why i must use functions instead of getters?